### PR TITLE
Govsi 868

### PR DESCRIPTION
**Pull request by sanisc**
## What?

- Align content for figma
- Update deployment variables for gov uk accounts publishing api

## Why?

To add final content to app.

### CODE REVIEW

File: src/locales/cy/translation.json [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
+    "serviceName": "Sign in to or create your GOV.UK account",
+    "serviceNameTitle": "GOV.UK account",
     "errorTitlePrefix": "Error",
     "back": "Back",
     "warning": "Warning",
     "errorSummaryTitle": "There is a problem",
     "phaseBanner": {
       "tag": "beta",
-      "content": "This is a new service – your {{-openLink}}feedback{{-closeLink}} will help us to improve it. // {{ target  }} //"
+      "text": "We’re trialling GOV.UK accounts - your <a  href=\"/feedback\" class=\"govuk-link\" rel=\"noreferrer noopener\" target=\"_blank\">feedback</a> your feedback will help us improve them."
```
Comment (chrisclayson) [inline src/locales/cy/translation.json:13]: You repeat "your feedback" here

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -2,24 +2,24 @@
...
+    "serviceName": "Sign in to or create your GOV.UK account",
+    "serviceNameTitle": "GOV.UK account",
     "errorTitlePrefix": "Error",
     "back": "Back",
     "warning": "Warning",
     "errorSummaryTitle": "There is a problem",
     "phaseBanner": {
       "tag": "beta",
-      "content": "This is a new service – your {{-openLink}}feedback{{-closeLink}} will help us to improve it. // {{ target  }} //"
+      "text": "We’re trialling GOV.UK accounts - your <a  href=\"/feedback\" class=\"govuk-link\" rel=\"noreferrer noopener\" target=\"_blank\">feedback</a> your feedback will help us improve them."
```
Comment (chrisclayson) [inline src/locales/en/translation.json:13]: And here :-)

File: src/locales/en/translation.json [hunk 3]
```diff
@@ -225,7 +230,7 @@
         "paragraph3": "The code will expire after  15 minutes.",
         "requestNewCodeLink": "Request a new code",
         "requestNewCodeParagraph": " if:",
-        "requestNewCodeReason1": "the code isn’t working or you didn’t receive one",
+        "requestNewCodeReason1": "the code isn’t working or has expired, or you didn’t receive one",
```
Comment (chrisclayson) [inline src/locales/en/translation.json:233]: Too many "or"s, should this be "isn't working, has expired or you didn't receive one"?

Comment (sanisc) [reply src/locales/en/translation.json:233]: taken from figma so i assume thats the truth

**Review by chrisclayson — APPROVED** [review]

**Pull request merged**